### DISCUSSION TITLE
공지사항

### DISCUSSION THREAD

**Discussion by cccwon2** [FE9-2/workroot#1] [Announcements]
일일 회의록을 쭉 정리해서 올릴께요.

**Comment by cccwon2** [comment]
# 2024-11-18 회의록

## 1. 검토한 프로젝트 후보

- 코워커스
- 알바폼
- 디지털노마드

## 2. 최종 선정: 알바폼

### 선정 이유

- 기본적인 기능 흐름이 명확함
- 초기 학습자 입장에서 접근하기 좋은 구조
- 기존 정적인 디자인을 개선할 여지가 많음
- 실사용자를 확보할 가능성이 있는 서비스

### 개선 및 차별화 방향

1. AI 기능 통합

   - 각종 AI API 기능 활용
   - AI 기반 채팅 상담 기능
   - 실시간 알바 채용 정보 제공

2. 디자인 개선

   - 현대적인 UI/UX 적용
   - 사용자 경험 개선

3. 기술 스택 검토사항
   - Three.js 등 모던 라이브러리 활용 검토
   - GitHub Actions CI/CD 구축

## 3. 다음 단계

- [x] 서비스 이름 결정 (AI 도메인 활용 검토)
- [ ] 전체적인 컨셉 구체화
- [ ] 기술 스택 확정
- [ ] 세부 기획 진행

## 4. 특이사항

- 커스텀한 백엔드 API 필요시 별도 개발 가능

**Comment by cccwon2** [comment]
# 2024-11-19 회의록

## 1. 프로젝트 진행 현황

### 1.1 도메인 구매 완료

- [[workroot.com](http://workroot.com/)](http://workroot.com/) (메인)
- [[workroute.com](http://workroute.com/)](http://workroute.com/) (개발)
- vercel 연동 완료 (fork한 개인 레포지토리 통해 연동)

### 1.2 Git 전략 논의

- Git Flow 대신 간소화된 GitHub Flow 채택
- Feature 브랜치 전략:
    - `feature/[기능명]` 형식으로 브랜치 생성
    - 기능별로 브랜치 분리 (One Feature, One Branch)
    - 예: `feature/modal`, `feature/hero` 등
- 배포 프로세스:
    - dev 브랜치에서 개발
    - 2-3주 간격으로 main 브랜치로 머지 예정

### 1.3 역할 분담 초안

- App Router 적용
- Auth (로그인/회원가입)
- Custom Hook 개발
- API 연동
- UI/컴포넌트 개발
- 디자인 시스템 구축

## 2. 기술 스택 관련

### 2.1 추가 예정 기술

- Storybook 적용 예정
- Husky 설정 완료
- CI/CD 구축 예정

### 2.2 디자인 시스템

- Primary 컬러: Orange → Green으로 변경
- 테마 시스템 구축 검토
- 테마 적용은 추후 고려

## 3. 프로젝트 관리

### 3.1 이슈 관리

- GitHub Issues를 통한 태스크 관리 예정
- 대략 50개 이상의 이슈 생성 후 분배 예정

### 3.2 회의록 관리

- AI 활용하여 회의록 작성
- Notion에 기록 후 Discussion에도 공유

## 4. 다음 단계

- [ ]  컬러 시스템 확정
- [ ]  로고 디자인 (DALL·E등 활용 검토)
- [ ]  GitHub Issues 생성
- [ ]  컴포넌트 분배

## 5. 특이사항

- 6주 프로젝트 기간 동안 경쟁력있는 포트폴리오로 활용 가능한 퀄리티를 추구
- 취업 준비생 관점에서 프로젝트 방향성 설정

**Reply by yyezzzy** [reply]
Daily Scrum 만들었어요 ~ 여기다 옮길게요 !
